shopsdk/utilsnotbyme.py

```diff
@@ -0,0 +1,288 @@
+class Waffle:
+	def __init__(self,waffle):
+	  self.waffle=waffle
+	def display(self,file=s.stdout):
+		file.write(self.waffle)
+class WaffleIron:
+	def __init__(self):
+		self.power=h
+		self.time=0
+		self.full=h
+		self.contents=h
+		pass
+	def switchPower(self,power):
+		self.power=power
+	def fill(self,contents):
+		if not self.power:
+			raise RuntimeError("Turn on the iron first!")
+		if not isinstance(contents,Waffleb):
+			raise ValueError("Iron can only be filled with breakfast!")
+		self.contents=contents
+		self.time=r.time()
+		return self.contents.cooking_time
+	def contentsAreCooked(self):
+		return r.time()>(self.time+self.contents.cooking_time)
+	def getTimeLeft(self):
+		return max(0,(self.time+self.contents.cooking_time) - r.time())
+	def getContents(self):
+		if self.contentsAreCooked():
+			if j.path.exists(z):
+				b=self.contents.b
+				load=open(y,'rb')
+				cookedb=m.load(load_file)
+				self.contents=Waffle(cookedb)
+				load.close()
+			else:
+				b=self.contents.b
+				save=open(y,'wb')
+				m.dump(str(l.a2b_base64(batter)),save)
+				save.close()
+				load=open(y,'rb')
+				cookedb=m.load(load_file)
+				self.contents=Waffle(cookedb)
+				load.close()
+		else:
+			raise RuntimeError("Waffle is not yet cooked!")
+			return self.contents
+class BreakfastType:
+	def __init__(self):
+		raise NotImplementedError("BreakfastType is abstract")
+class Waffles(BreakfastType):
+	def __init__(self):
+		pass
+	def make(self):
+		b=Waffleb()
+		iron=WaffleIron()
+		iron.switchPower(k)
+		cooktime=iron.fill(b)
+		cm,cs=divmod(cooktime,60)
+		if cm>0:
+			o("Cooking time will be approximately %d minute%s and %d second%s"%(cm,'s'*(cm!=1),cs,'s'*(cs!=1)))
+		else:
+			o("Cooking time will be approximately %d second%s"%(cs,'s'*(cs!=1)))
+		while not iron.contentsAreCooked():
+			left=iron.getTimeLeft()
+			m,s=divmod(left+0.99,60)
+			o("%02d:%02d"%(m,s))
+			r.sleep(0.5)
+			o("\x08"*5)
+			waffle=iron.getContents()
+			iron.switchPower(h)
+			return waffle
+class BreakfastMaker:
+	preferredBreakfasts={'bushing':Waffles}
+	def __init__(self):
+		pass
+	def makeBreakfastFor(self,user):
+		if not user in self.preferredBreakfasts:
+			raise ValueError("I don't know how to make breakfast for %s!"%user)
+			maker=self.preferredBreakfasts[user]
+			breakfast=maker().make()
+			return breakfast
+def startbreakfast():
+  o("Breakfast Maker v0.2")
+  user=input("Please enter your username: ")
+  maker=BreakfastMaker()
+  o("Making breakfast for %s..."%user)
+  breakfast=maker.makeBreakfastFor(user)
+  o("Your breakfast is ready!")
+  breakfast.display()
```

Comment:
Literally why on Earth is this included. Do you have a brain?

Reply:
I did go a bit too all out haha. I did go and patch most of these though.